routes/api/contacts.js

```diff
@@ -1,25 +1,113 @@
-const express = require('express')
+const contactsMethots = require("./contactsMethots.js");
+const express = require("express");
+const router = express.Router();
+const Joi = require("joi");
 
-const router = express.Router()
+const schema = Joi.object({
+  name: Joi.string().alphanum().min(3).max(30).required(),
+  email: Joi.string().email({
+    minDomainSegments: 2,
+    tlds: { allow: ["com", "net"] },
+  }),
+  phone: Joi.string().alphanum().min(7).max(12).required(),
+  favorite: Joi.boolean().required(),
+}).with("username", "birth_year");
```

Comment:
`.with("username", "birth_year");` ta część do wyrzucenia. To znaczy że jeśli podasz pole username to musisz również podać pole birth_year co w naszym przypdaku nie ma sensu.